test/controllers/tickets_controller_test.rb

```diff
@@ -70,6 +71,18 @@ class TicketsControllerTest < ActionController::TestCase
 
   end
 
+  test 'should accept tickets in Base64 encoding' do
+    I18n.locale = :nl
+
+    assert_difference 'ActionMailer::Base.deliveries.size', User.agents.count do
+      assert_difference 'Ticket.count' do
+        post :create, message: @simple_base64_email, format: :json
```

Comment:
Nope, it's not particularly necessary - just thought it might be a useful addition. I can remove that and just keep the flag if that's better?